api/sql/userSQL/initialize.sql

```diff
@@ -1,9 +1,18 @@
 DROP TABLE IF EXISTS Users CASCADE;
+DROP TABLE IF EXISTS UserSessions CASCADE;
 
 CREATE TABLE Users (
   id SERIAL PRIMARY KEY,
   username TEXT NOT NULL,
   password TEXT NOT NULL,
   data_id INT NOT NULL,
   FOREIGN KEY (data_id) REFERENCES UserData (id)
+);
+
+CREATE TABLE UserSessions (
+  id SERIAL PRIMARY KEY,
+  session_key VARCHAR (100) NOT NULL,
+  user_id INT NOT NULL,
+  exp_date TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP + interval '4 hours',
```

Comment:
What does the `+ interval '4 hours'` do?

Reply:
this just adds four hours to the current timestamp. its similar timedelta() in python. I wasn't sure how long we want session tokens to be active, we can change this if need be